src/Components/ProductList/ProductList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import axios from 'axios';
+import OfferBox from 'components/OfferBox/OfferBox';
```

Comment:
teraz katalog `components` jest z wielkiej litery - zmienisz na małą? Tylko musisz to zrobić w 2 krokach, zmienić nazwę na inną - np. acomponents, a potem w 2 kroku zmienić na wielką, bo z tego co pamiętam windows się gubi :) A mi to wywala błędy jak odpalam :)

Reply:
pokombinuję z tym jeszcze, bo widzę, że nic się nie zmieniło

Reply:
Ja widzę tutaj, że gothub łapie Components zamiast z małej:) możliwe, że musisz to zmienić w 2 commitach - pierwszy z jakąkolwiek nazwa i 2 z dobrą :)

Reply:
udało się :)